examples/nlp/language_modeling/megatron_ckpt_to_nemo.py

```diff
@@ -92,6 +97,14 @@ def get_args():
     )
     parser.add_argument("--local_rank", type=int, required=False, default=os.getenv('LOCAL_RANK', -1))
     parser.add_argument("--bcp", action="store_true", help="Whether on BCP platform")
+    parser.add_argument(
+        "--precision",
+        type=str,
+        required=False,
+        default='16-mixed',
+        choices=['16-mixed', 'bf16-mixed'],
+        help="Precision value for the trainer that matches with precision of the ckpt",
+    )
```

Comment:
shouldn't 32 be here as well?

Reply:
Yes I can add that, but I was thinking we'll be just using fp16 or bf16 for training.

Reply:
Added. Also `punctuation_capitalization_train_evaluate.py` needed `ddp_find_unused_parameters_true` to be added, added that in this PR as well.